### PR TITLE
Npe2 fix precommit

### PR DESCRIPTION
**Pull request by nclack**
precommit wasn't getting applied bc the logic around the if statement wasn't quite right.

This also fixes a typo in the setup.cfg

### CODE REVIEW

File: {{cookiecutter.plugin_name}}/setup.cfg [hunk 1]
```diff
@@ -74,7 +74,7 @@ where = src
 
 [options.package_data]
 {{cookiecutter.plugin_name}} = 
-    napari.yaml
+    napari.yml
```
Comment (tlambert03) [inline {{cookiecutter.plugin_name}}/setup.cfg:77]: we should decide what convention we're going to use. I've generally been going with `napari.yaml`.  YAML [prefers](https://yaml.org/faq.html) we use `.yaml` over `.yml`,  but we should just be consistent

**Review by tlambert03 — APPROVED** [review]
thanks!

turned off white space for reviewing... but could you perhaps turn off your auto-formatter for markdown?

File: {{cookiecutter.plugin_name}}/setup.cfg (hunk 1 above)
Comment (nclack) [reply {{cookiecutter.plugin_name}}/setup.cfg:77]: I'm ok with just deciding on `.yaml`

Comment (nclack) [reply {{cookiecutter.plugin_name}}/setup.cfg:77]: >  auto-formatter for markdown?

I will do my best :) did that affect this pr?

**Comment by nclack** [comment]
>  auto-formatter for markdown?

I will do my best :) did that affect this pr?

**Comment by tlambert03** [comment]
> > auto-formatter for markdown?
> 
> I will do my best :) did that affect this pr?

ah, it did when I reviewed it!  It had ~250 lines changed... now it's down to 5, so maybe there was a merge in the meantime?

**Comment by nclack** [comment]
> maybe there was a merge in the meantime

there was not, but looking at it, it looks like I screwed up the upstream merge.

**Pull request merged**